third_party/go/BUILD

```diff
@@ -1,7 +1,82 @@
+go_toolchain(
```

Comment:
What do you think about writing the pex tool in python at some point so we can avoid depending on go here?

Reply:
Added to my to-do list. Will have a look into it.